src/convertToCase/inputValidation.js

```diff
@@ -0,0 +1,33 @@
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+function inputValidation(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
```

Comment:
You're checking here if the `originalText` is not empty, but it might be a good idea to also check if it's a string. This can help prevent errors if a client sends a request with a non-string value for `originalText`.